config/settings.yml

```diff
@@ -834,6 +834,13 @@
     - MoveIntoResourcePool
     :VmSuspendedEvent:
     - SuspendVM_Task
+:git_repository_proxy:
+  :host:
+  :password:
+  :port:
+  :user:
+  :scheme:
+  :path:
```

Comment:
proxies can have a path?

Reply:
This was just in response to this error:

```
irb(main):003:0> Rugged::Repository.clone_at("https://github.com/ManageIQ/container-ruby", "./things", :proxy_url => "http://example.com")
Traceback (most recent call last):
       13: from bin/rails:4:in `<main>'
       12: from bin/rails:4:in `require'
       11: from railties (5.1.7) lib/rails/commands.rb:16:in `<top (required)>'
       10: from railties (5.1.7) lib/rails/command.rb:44:in `invoke'
        9: from railties (5.1.7) lib/rails/command/base.rb:63:in `perform'
        8: from thor (0.19.4) lib/thor.rb:369:in `dispatch'
        7: from thor (0.19.4) lib/thor/invocation.rb:126:in `invoke_command'
        6: from thor (0.19.4) lib/thor/command.rb:27:in `run'
        5: from railties (5.1.7) lib/rails/commands/console/console_command.rb:97:in `perform'
        4: from railties (5.1.7) lib/rails/commands/console/console_command.rb:17:in `start'
        3: from railties (5.1.7) lib/rails/commands/console/console_command.rb:62:in `start'
        2: from (irb):3
        1: from (irb):3:in `clone_at'
Rugged::NetworkError (invalid url, missing path)
```